hermetic_build/common/requirements.txt

```diff
@@ -0,0 +1,109 @@
+#
+# This file is autogenerated by pip-compile with Python 3.11
```

Comment:
What are the implications of generating a reqs file with a different python version? Maybe we can create one with 3.12 instead?

Reply:
`pip-tools` (which contains `pip-compile`) is built with python 3.11.

You can confirm this by running `which pip-compile`.

Reply:
Ah, looks like pip-compile is available only for 3.11, but it's installable via `pip install pip-tools` from 3.12 onwards. I managed to locally generate the hashes with python 3.12. Can you confirm this works?

```bash
pyenv install 3.12
pyenv local 3.12
python --version
pip install pip-tools
pip-compile --generate-hashes
```

Reply:
Thanks.

I changed the `requirements.txt` using python 3.12.